admin: Poll voting

BeebBeebBoob opened the pull request:
<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание

<!-- Опишите, что делает ваш Pull request. Документировать каждую деталь не требуется, просто укажите основные изменения. -->

## Причина создания ПР / Почему это хорошо для игры

<!-- Здесь можно оставить ссылку на сообщение в #отчеты-по-предложениям, чтобы подтвердить, что предложение обсуждалось внутри Discord-сообщества. -->
<!-- Если отчёта нет, то укажите, почему это изменение положительно влияет на игру. -->
<!-- В случае исправления бага, укажите ссылку на канал в #баг-репорты-v2 или issue в репозитории. В ином случае, опишите баг и ступени для его воспроизведения. -->
<!-- Пример ссылки в Discord-сообщество : https://discord.com/channels/617003227182792704/755125334097133628/ID-сообщения -->

## Демонстрация изменений

<!-- В случае наличия изменений, влияющих на игровую часть, опишите их здесь. В случае их отсутствия, этот пункт можно удалить -->

## Тесты

<!-- Здесь необходимо описать шаги, которые предпринимались для тестирования изменения. Этот пункт обязателен, без него Pull request будет рассматриваться дольше. -->

BeebBeebBoob committed:
add: Poll voting

PlayerUnknown14 commented on code/controllers/subsystem/non-firing/titlescreen.dm:
```diff
@@ -278,6 +278,7 @@ SUBSYSTEM_DEF(title)
 		<a class="menu_button" href='byond://?src=[player.UID()];game_preferences=1'>Настройки игры</a>
 		<hr>
 		<a class="menu_button" href='byond://?src=[player.UID()];sound_options=1'>Настройки громкости</a>
+		<a class="menu_button" href='byond://?src=[player.UID()];poll_panel=1'>Открыть голосование</a>
```
```suggestion
		<a class="menu_button" href='byond://?src=[player.UID()];poll_panel=1'>Открыть текущее голосование</a>
```

PlayerUnknown14 reviewed (changes requested):
Да.

PlayerUnknown14 commented:
Не понял, от меня как главлока здесь что-то конкретное по локализации требуется или просто апрув накинуть и всё?

BeebBeebBoob committed:
bundle update

BeebBeebBoob commented:
Протупел чёт. Я 3 дня это клепал и уже без интеллекта остался

BeebBeebBoob committed:
Update code/controllers/subsystem/non-firing/titlescreen.dm

Co-authored-by: PlayerUnknown14 <139672964+PlayerUnknown14@users.noreply.github.com>

PlayerUnknown14 reviewed (dismissed):
Дерьмо случается.

BeebBeebBoob committed:
ngggh I'll do later. Fixes polling mostly and TGUI. but has SQL issues
Column 'createdby_ip' cannot be null INSERT INTO poll_question
{ ERROR 1048 (23000): Column 'pollid' cannot be null

BeebBeebBoob committed:
Final fixes and updates

BeebBeebBoob committed:
SQL OOPS ALL MY VERSIONS TO 34
